src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        int ball1 = lottery.getRandomBall();
+        int ball2 = lottery.getRandomBall();
+        int ball3 = lottery.getRandomBall();
+
+        System.out.println(ball1);
+        System.out.println(ball2);
+        System.out.println(ball3);
     }
```

Comment:
Creating multiple instances of the same class when a single instance can be reused is not efficient. Here, you are creating three separate balls but you could reuse the same `Lottery` instance to get multiple random balls.